ptbcontrib/roles/roleshandler.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains the RoleHandler class."""
+from typing import Any
+
+from telegram import Update
+from telegram.ext import Handler, CallbackContext, Dispatcher
+
+from .roles import Role, Roles
+
+BOT_DATA_KEY: str = 'ptbcontrib_roles_bot_data_key'
```

Comment:
I think it should be public. Thing is that storing stuff in `bot_data` can cause problems if people use that key for other things, so it's not abstraction enough to go for hiding it instead of documenting, imo.